app/src/main/java/com/door43/translationstudio/core/TargetTranslationMigrator.java

```diff
@@ -347,7 +346,66 @@ private static boolean v2( File path) throws Exception {
         return true;
     }
 
-   /**
+    /**
+     * convert projects from USX to USFM
+     * @param targetTranslations
+     * @return
+     */
+    public static boolean migrateFromUSXtoUSFM(TargetTranslation[] targetTranslations) {
```

Comment:
These methods as well as the other methods for merging chunks should not be public. They should instead be ran internally for the appropriate target translation manifest version. Calling these public static methods elsewhere makes it hard to maintain the migration code.